mmdet/apis/train.py

```diff
@@ -114,13 +115,15 @@ def train_detector(model,
         find_unused_parameters = cfg.get('find_unused_parameters', False)
         # Sets the `find_unused_parameters` parameter in
         # torch.nn.parallel.DistributedDataParallel
-        model = MMDistributedDataParallel(
-            model.cuda(),
-            device_ids=[torch.cuda.current_device()],
+        model, DDP = build_ddp(model, cfg.device)
```

Comment:
It would be better if the building process in lines 119-123 can be done in build_ddp so that build_ddp only need to return the wrapped model

Reply:
> It would be better if the building process in lines 119-123 can be done in build_ddp so that build_ddp only need to return the wrapped model

good suggestion, we've done it